### PR TITLE
refine postgres migration scritpts

### PR DESCRIPTION
**Pull request by lziest**
per @benburkert's suggestion, refining postgres sql migration.

### CODE REVIEW

**Comment by kisom** [comment]
LGTM.

File: certdb/pg/migrations/001_CreateCertificates.sql [hunk 1]
```diff
@@ -3,16 +3,25 @@
 
 CREATE TABLE certificates (
   id                serial,
```
Comment (benburkert) [inline certdb/pg/migrations/001_CreateCertificates.sql]: `serial primary key`

File: certdb/pg/migrations/001_CreateCertificates.sql [hunk 2]
```diff
@@ -3,16 +3,25 @@
...
+  pem               bytea NOT NULL,
+  PRIMARY KEY(id, serial)
 );
 
+CREATE TABLE ocsp_responses (
+  id                serial PRIMARY KEY,
+  serial            bytea NOT NULL,
+  body              bytea NOT NULL,
+  expiry            timestamptz,
+  FOREIGN KEY(serial) REFERENCES certificates(serial)
```
Comment (benburkert) [inline certdb/pg/migrations/001_CreateCertificates.sql]: it's probably better to do the foreign key on `certificate.id`:
`certificate_id integer REFERENCES certificates (id)`

File: certdb/pg/migrations/001_CreateCertificates.sql (hunk 1 above)
Comment (lziest) [inline certdb/pg/migrations/001_CreateCertificates.sql]: I made (id, serial) primary key, would that works as well?

File: certdb/pg/migrations/001_CreateCertificates.sql (hunk 2 above)
Comment (lziest) [inline certdb/pg/migrations/001_CreateCertificates.sql]: hmm, what if we want to flush all ocsp responses manually and run a OCSP refresh? sounds like this foreign key on id would make it impossible?

**Comment by lziest** [comment]
need to address #503, close this.

**Pull request closed without merging**